Code/Mantid/Framework/DataHandling/src/SaveOpenGenieAscii.cpp

```diff
@@ -6,11 +6,14 @@
 #include "MantidKernel/ListValidator.h"
 #include "MantidKernel/Exception.h"
 #include "MantidKernel/Property.h"
+#include "MantidKernel/TimeSeriesProperty.h"
 #include <Poco/File.h>
 #include <Poco/Path.h>
 #include <fstream>
 #include <exception>
 
+#include <vector>
```

Comment:
Remember the rule to try to keep headers sorted alphabetically (here first the MantidKernel/... headers, and then the standard library headers). This is not really that important in this particular file but can very helpful in general.